assayist/processor/container_go_analyzer.py

```diff
@@ -0,0 +1,139 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import os
+import subprocess
+
+from assayist.common.models import content, source
+from assayist.processor.base import Analyzer
+from assayist.processor.logging import log
+
+
+class ContainerGoAnalyzer(Analyzer):
+    """Analyze the Go executables in a container image layer."""
+
+    # goversion executable path.
+    GOVERSION = 'goversion'
+
+    # Backvendor executable path.
+    BACKVENDOR = 'backvendor'
+
+    # Output format tempate for backvendor.
+    BACKVENDOR_TEMPLATE = '\t{{.Ver}}\t{{.Repo}}\t{{.Rev}}'
```

Comment:
What's the point of the `\t` prefix?

Also, typo in `template` in comment above.

Reply:
Fixed typo and updated comment to explain how template is used.